src/postprocess/stresses.jl

```diff
@@ -21,8 +21,8 @@ Compute the stresses (= 1/Vol dE/d(M*lattice), taken at M=I) of an obtained SCF
                                    basis.kgrid, basis.kshift, basis.symmetries_respect_rgrid,
                                    basis.comm_kpts)
         ρ = DFTK.compute_density(new_basis, scfres.ψ, scfres.occupation)
-        energies, _ = energy_hamiltonian(new_basis, scfres.ψ, scfres.occupation;
-                                         ρ, scfres.eigenvalues, scfres.εF)
+        energies = energy_hamiltonian(new_basis, scfres.ψ, scfres.occupation;
+                                      ρ, scfres.eigenvalues, scfres.εF).E
```

Comment:
This I find confusing. We could rename `E` in `energy_hamiltonian` to `energies` though.